lib/linter.js

```diff
@@ -1197,10 +1197,11 @@ class Linter extends EventEmitter {
      * @param {string} options.filename The filename from which the text was read.
      * @param {boolean} options.allowInlineConfig Flag indicating if inline comments
      *      should be allowed.
+     * @param {boolean|Function} shouldFix Determines whether fixes should be applied
      * @returns {Object} The result of the fix operation as returned from the
      *      SourceCodeFixer.
      */
-    verifyAndFix(text, config, options) {
+    verifyAndFix(text, config, options, shouldFix) {
```

Comment:
Nitpick: Could `fix` be one of the options (third parameter) rather than adding another parameter to this method (e.g. as a `fixFilter` property)? It would be nice to avoid the [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap).